pkg/program/shell/runtime.go

```diff
@@ -72,10 +73,13 @@ func InitRuntime(binpath, sockpath, dbpath string) (*eval.Evaler, string) {
 
 	ev := eval.NewEvaler()
 	ev.SetLibDir(filepath.Join(dataDir, "lib"))
+
 	ev.InstallModule("math", eval.MathNs)
+	ev.InstallModule("platform", platform.Ns)
 	ev.InstallModule("re", re.Ns)
 	ev.InstallModule("str", str.Ns)
-	ev.InstallModule("platform", platform.Ns)
+	ev.InstallModule("unix", unix.Ns)
```

Comment:
Don't install this namespace at all for non-UNIX operating systems.

You can do this by declaring a constant in the `unix` package, whose value is true on UNIX but false on others.

Reply:
Okay, but it seemed cleaner to me to unconditionally install an empty namespace rather than predicate this statement on whether we're actually building on a UNIX-like system. In fact, if I do this why would we need a `unix_non_unix.go` that declares an empty namespace?

Reply:
You need a `unix_non_unix.go` to make the code compile :)

My preference is to have a **Go** `unix` package with an empty `Ns` variable on non-Unix OS, and to not install the **Elvish** `unix` module on non-Unix OS.

Reply:
Done